modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in the FIO namespace. After the objects are removed, you can then remove the Operator and its namespace by deleting the `FIM` resources.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The File Integrity Operator is installed.
+
+.Procedure
+
+To remove the File Integrity Operator by using the {product-title} web console:
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+
+. Delete all `FileIntegrity` objects.
+
+. Switch to the *Administration* -> *Operators* -> *Installed Operators* page.
+
+. Click the Options menu {kebab} on the *File Integrity Operator* entry and select *Uninstall Operator*.
+
+. Switch to the *Home* -> *Projects* page.
+
+. Search for `openshift-file-integrity`.
+
+. Click the Options menu {kebab} next to the *openshift-file-integrity* project, and select *Delete Project*.
+
+. Confirm the deletion by typing `` in the `openshift-file-integrity` dialog box, and click *Delete*.
+
+. Delete the `MutatingWebhookConfiguration` object by running the following command:
```

Comment:
File Integrity Operator will not create any `MutatingWebhookConfiguration` object. Please remove this step.